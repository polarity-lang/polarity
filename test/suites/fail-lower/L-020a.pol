data Bar { B }
note Foo
let x: Bar { Foo }
