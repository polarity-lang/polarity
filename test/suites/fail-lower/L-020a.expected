L-020

  × Invalid usage of note Foo
   ╭─[L-020a.pol:3:14]
 2 │ note Foo
 3 │ let x: Bar { Foo }
   ·              ───
   ╰────
